### DISCUSSION TITLE
How to have two email addresses, one private, the other public?

### DISCUSSION THREAD

**Discussion by duritia** [community/community#19448] [Other Feature Feedback, Questions, & Ideas] [answered]
How to have two email addresses, one private, the other public? The option available seems to turn them all public or private, without means to choose other options.

**Comment by doguedogue** [comment, marked as answer]
Here you define your public email (if you want): [https://github.com/settings/profile](https://github.com/settings/profile)

In this place you add all the emails you want private and set the one by the default: [https://github.com/settings/emails](https://github.com/settings/emails)

**PD**: Please mark your question as answered if you feel that it was.

**Reply by duritia** [reply]
Thank you very much for your explanation, doguedogue. I had visited these 2 pages, and the problem is in your second link. It has a checkbox that says  "Keep my email addresses private", in plural:

![image](https://user-images.githubusercontent.com/69221194/175818990-264f0ade-158f-43df-aed2-0906572ed24b.png)

So, if we make only one address public, and the rest are kept private (as i expect to have), this needs to be changed in Github, in my opinion. Further, the documentation found by me does not say what you tell here. I will trust you, however.

Again, thank you very much.
